### PR TITLE
Move tests to Github Actions

### PR DESCRIPTION
**Pull request by guilhermef**
Moving tests from TravisCI to Github Actions

### CODE REVIEW

**Comment by guilhermef** [comment]
@scorphus 
image

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:11]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+name: "CodeQL"
+
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:13]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:18]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    branches: [ master ]
+  schedule:
+    - cron: '37 1 * * 3'
+
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:22]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:31]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'python' ]
+        # CodeQL supports [ 'cpp', 'csharp', 'go', 'java', 'javascript', 'python', 'ruby' ]
+        # Learn more about CodeQL language support at https://git.io/codeql-language-support
+
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:38]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    # Initializes the CodeQL tools for scanning.
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:43]:
```suggestion
```

File: .github/workflows/codeql-analysis.yml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+    #- name: Autobuild
+    #  uses: github/codeql-action/autobuild@v1
+
+    # ℹ️ Command-line programs to run using the OS shell.
+    # 📚 https://git.io/JvXDl
+
+    # ✏️ If the Autobuild fails above, remove it and uncomment the following three lines
+    #    and modify them (or add more) to build your code if your project
+    #    uses a compiled language
+
```
Comment (scorphus) [inline .github/workflows/codeql-analysis.yml:64]:
```suggestion
```

**Review by scorphus — APPROVED** [review]
Pleaes, consider the suggestions below a bunch of nits 🙂

File: .github/workflows/stale.yml [hunk 9]
```diff
@@ -0,0 +1,40 @@
+---
+# This workflow warns and then closes issues and PRs that have had no
+# activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
```
Comment (scorphus) [inline .github/workflows/stale.yml:7]:
```suggestion
```

File: .github/workflows/stale.yml [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+name: Close stale issues and PRs
+on:
+  schedule:
+    - cron: '* */12 * * *'
+
```
Comment (scorphus) [inline .github/workflows/stale.yml:12]:
```suggestion
```

**Review by scorphus — APPROVED** [review]

**Comment by guilhermef** [comment]
Thanks @scorphus 
Brad pitt

**Comment by scorphus** [comment]
Thank you, @guilhermef. This was a nice one.

**Pull request merged**